internal/commands/validate/validate.go

```diff
@@ -89,6 +90,10 @@ func validate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if shouldPrint {
```

Comment:
Consider moving this block before the parsing is done, so that when the `cirrus validate --print` is invoked with an invalid config, the parsing errors are printed after the potentially huge configuration, so it's easier to find them in the terminal output.